AzureAppServiceSetAppSettings/Tasks/AppSettings/ApplyAppSettings.ps1

```diff
@@ -10,7 +10,7 @@ param(
     $ResourceGroupName,
 
     [String] [Parameter(Mandatory = $false)]
-    $Slot,
+    $Slot = "",
```

Comment:
Isn't it better to use "production" as the default value of the $Slot parameter? Then you won't need the check and reassignment at line 25-27.

Reply:
I was under the impression that the $Slot parameter would always be passed the textbox's value from the extension form in VSTS. This means that if the user leaves the Slot textbox empty, an empty string will be passed to the $Slot parameter, which will override the parameter's default value. This is why I used an If to set $Slot to "production" when empty. This would mean that the default "" value here is actually useless, I think. Does it make sense?

Reply:
Ah you're right, makes total sense! 👍 For some reason I was under the impression the PowerShell script would treat an empty string as no input. I don't know why to be honest haha (brain short circuit maybe 😆).

Reply:
arf yes !